app/main.cpp

```diff
@@ -11,25 +11,57 @@
 #include "TranslationModelConfiguration.h"
 #include "TranslationRequest.h"
 #include "TranslationResult.h"
+#include "translator/parser.h"
 
 int main(int argc, char **argv) {
 
-  // Create an instance of AbstractTranslationModel with a dummy model
-  // configuration
-  TranslationModelConfiguration config(
-      "dummy_modelFilePath", "dummy_sourceVocabPath", "dummy_targetVocabPath");
+  // Create a configParser and load command line parameters into a YAML config
+  // string.
+  auto configParser = marian::bergamot::createConfigParser();
+  auto options = configParser.parseOptions(argc, argv, true);
+  std::string config = options->asYamlString();
+  std::cout << config << std::endl;
```

Comment:
Please remove `std::cout << config << std::endl;` from here.

Reply:
Covered in https://github.com/browsermt/bergamot-translator/pull/8/commits/e76a602dc7205567fdcb76820349bafa8f51bf51. Also removes a file-path in output, enabling a test for bergamot-translator-app in [browsermt/bergamot-translator-tests](https://github.com/browsermt/bergamot-translator-tests).